tests/parametric/test_dynamic_configuration.py

```diff
@@ -131,7 +131,7 @@ def test_apply_state(self, library_env, test_agent, test_library):
         This signal, along with the telemetry event, is used to determine when the
         configuration has been applied by the tracer.
         """
-        set_and_wait_rc(test_agent, {})
+        set_and_wait_rc(test_agent, config_overrides={"tracing_sampling_rate": 0.5})
```

Comment:
The default value for trace sample rate in Go is nil, so I needed to set a different value here to trigger a config change telemetry event. Lmk if there is a better way to do this.